blackjax/util.py

```diff
@@ -281,6 +281,10 @@ def transform(state_and_incremental_val, info):
     return SamplingAlgorithm(init_fn, update_fn), transform
 
 
+def safediv(x, y):
```

Comment:
is this really make a differences?

Reply:
Yes, because there was a 0/0 case. It could be avoided other ways, but this was the cleanest.